tests/phpunit/includes/object-cache.php

```diff
@@ -977,20 +977,20 @@ public function add( $key, $value, $group = 'default', $expiration = 0, $server_
 		}
 
 		$derived_key = $this->buildKey( $key, $group );
-		$expiration  = $this->sanitize_expiration( $expiration );
+
+		// Add does not set the value if the key exists; mimic that here.
+		if ( isset( $this->cache[ $derived_key ] ) ) {
+			return false;
+		}
 
 		// If group is a non-Memcached group, save to runtime cache, not Memcached.
 		if ( in_array( $group, $this->no_mc_groups, true ) ) {
 
-			// Add does not set the value if the key exists; mimic that here.
-			if ( isset( $this->cache[ $derived_key ] ) ) {
-				return false;
-			}
-
 			$this->add_to_internal_cache( $derived_key, $value );
 
 			return true;
 		}
+		$expiration = $this->sanitize_expiration( $expiration );
```

Comment:
```suggestion

		$expiration = $this->sanitize_expiration( $expiration );
```
Just adding an empty line to separate from the conditional above.